content-samples/react/emails/notifications/novu-credit-alert.tsx

```diff
@@ -0,0 +1,281 @@
+      </Body>
+    </Html>
+  );
+};
+
+const main = {
+  backgroundColor: '#f3f4f6',
+  fontFamily: '-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,"Helvetica Neue",Ubuntu,sans-serif',
+};
+
+const container = {
+  backgroundColor: '#ffffff',
+  margin: '0 auto',
+  padding: '24px',
+  marginBottom: '24px',
+  maxWidth: '600px',
+  borderRadius: '8px',
+  boxShadow: '0 1px 3px 0 rgba(0, 0, 0, 0.1), 0 1px 2px 0 rgba(0, 0, 0, 0.06)',
+};
+
+const iconContainer = {
+  textAlign: 'center' as const,
+};
+
+const h1 = {
+  color: '#1f2937',
+  fontSize: '24px',
+  fontWeight: 'bold',
+  textAlign: 'center' as const,
+  margin: '16px 0 24px',
+};
+
+const h2 = {
+  color: '#1f2937',
+  fontWeight: 'bold',
+  fontSize: '20px',
+  verticalAlign: 'middle'
+};
+
+const text = {
+  color: '#4b5563',
+  fontSize: '16px',
+  lineHeight: '24px',
+};
+
+const icon: React.CSSProperties | undefined = {
+    width: "48",
+    height: "48",
+    fill: "none",
+    stroke: "currentColor",
+    strokeWidth: "2",
+    strokeLinecap: "round",
+    strokeLinejoin: "round",
+    color: '#DA3688',
+    marginBottom: '16px',
+};
+
+const infoBox: React.CSSProperties | undefined = {
+  backgroundColor: '#FFF2FA',
+  border: '1px solid #FF968A',
+  borderRadius: '8px',
+  padding: '16px',
+  marginBottom: '24px',
+};
+
+const infoIcon = {
+    marginRight: '8px',
+    color: '#DA0685',
+    verticalAlign: 'middle'
+};
+
+const progressBar = {
+  width: '100%',
+  backgroundColor: '#e5e7eb',
+  borderRadius: '9999px',
+  height: '16px',
+  marginBottom: '8px',
+  marginTop: '8px'
+};
+
+const progressFill = {
+  backgroundColor: '#DA0685',
+  height: '16px',
+  borderRadius: '9999px',
+};
+
+const button = {
+  backgroundColor: '#3b82f6',
+  borderRadius: '4px',
+  color: '#ffffff',
+  fontSize: '16px',
+  fontWeight: 'bold',
+  textDecoration: 'none',
+  textAlign: 'center' as const,
+  display: 'block',
+  padding: '12px 16px',
+};
+
+const hr = {
+  borderColor: '#e5e7eb',
+  margin: '20px 0',
+};
+
+const footer = {
+  color: '#6b7280',
+  fontSize: '14px',
+  lineHeight: '24px',
+};
+
+const link = {
+  color: '#3b82f6',
+  textDecoration: 'none',
+};
+
+const list = {
+  ...text,
+  paddingLeft: '24px',
+  margin: '0 0 16px',
+};
+
+const highlight = {
+  color: '#DA3688',
+  fontWeight: 'bold',
+};
+
+NovuCreditUsageAlertEmail.PreviewProps = {
+    companyName: "Novu",
+    logoImg: `https://images.spr.so/cdn-cgi/imagedelivery/j42No7y-dcokJuNgXeA0ig/dca73b36-cf39-4e28-9bc7-8a0d0cd8ac70/standalone-gradient2x_2/w=128,quality=90,fit=scale-down`,
+    address: '',
+    linkToPricing: "https://novu.co/pricing/?utm_campaign=docs_top_nav",
+    linkToUnsubscribe: "https://google.com",
+    customerName: "Mussi",
+    usage: 80,
+    amount: 130,
+    packagesList: [{
+        name: 'BUSINESS',
+        amount: '250K events/month',
+        price: '$250'
+    },
+    {
+        name: 'ENTERPRISE',
+        amount: '5M events/month',
+        price: `Let's Talk`
```

Comment:
I think there might be a mistake here?

Reply:
This is not a mistake. In the case of a large client, there is no fixed price, and it is determined individually based on the specific opportunity.